addons/web/static/tests/core/hotkeys/hotkey_service_tests.js

```diff
@@ -181,6 +194,44 @@ QUnit.test("non-MacOS usability", async (assert) => {
     removeHotkey();
 });
 
+QUnit.test("the overvay of hotkeys is correctly displayed", async (assert) => {
+    assert.expect(7);
+
+    class MyComponent extends Component {
+        onClickB() {
+            assert.step("click b");
+        }
+        onClickC() {
+            assert.step("click c");
+        }
```

Comment:
FYI, you could also do that
```suggestion
onClick(ev) {
    assert.step(`click ${ev.target.dataset.hotkey}`);
}
```